src/bundler.ts

```diff
@@ -79,7 +79,7 @@ class Bundler {
                     String(opts.abspath).trim() !== '') ?
         path.resolve(opts.abspath) :
         undefined;
-    this.pathResolver = new PathResolver(Boolean(this.abspath));
+    this.pathResolver = new PathResolver(this.abspath);
```

Comment:
What's the use case for the first option? It seems the same as `<base>`, but I'm not sure when we'd want to only affect relative URLs, not all URLs.

URLResolvers, as you note, have nearly complete control over the URL space, but implicitly (and we should make this more clear if necessary), it's expected that all URLs are resolved to the base URL of their enclosing document first, making them effectively all absolute anyway.

We definitely need to be able to prefix all of the absolute URLs to enable mounting an app at a subfolder. We've had this requested a bunch (I can't find the issue in polymer-cli or polymer-build, @FredKSchott ?). I'm not sure that bundler is the place for it, as opposed to polymer-build. In fact, it seems like the only path munging that the bundler should be doing is the minimal necessary because it's moving content to different document URLs.

If that's not the only thing that PathResolver does, what else is it doing and can we get rid of it?